Update about.md (#1388)
usegalaxy-eu/website: about.md

Our Galaxy server ([https://usegalaxy.eu](https://usegalaxy.eu)) is the **biggest Galaxy instance in Europe**, and one of the biggest worldwide.

With this server, we provide access to:
- a huge compute and storage resource free of charge,
- more than 3900 different, well-documented and constantly maintained scientific tools,

> - 250+ GB per user (500 GB for ELIXIR members).

Reviewer: ```suggestion
- 250+ GB per user (500 GB for ELIXIR members).
- We also support numerous additional storage options and you can learn about them [here](https://galaxyproject.org/eu/storage/).
```

  Individuals who have contributed to the project, but whose contributions do not rise to the level justifying authorship, can be recognized in the acknowledgements section of the manuscript as follows:

>   > The authors acknowledge the support of the Freiburg Galaxy Team: Person *X* and Björn Grüning, Bioinformatics, University of Freiburg (Germany), funded by the German Federal Ministry of Education and Research [BMBF](http://www.bmbf.de/) grant 031 A538A [de.NBI](https://www.denbi.de)-RBC and the Ministry of Science, Research and the Arts Baden-Württemberg (MWK) within the framework of LIBIS/de.NBI Freiburg.

Reviewer: ```suggestion
  > The authors acknowledge the support of the Freiburg Galaxy Team: Person *X* and Björn Grüning, Bioinformatics, University of Freiburg (Germany), funded by the German Federal Ministry of Education and Research [BMFTR](https://www.bmftr.bund.de) grant 031 A538A [de.NBI](https://www.denbi.de)-RBC and the Ministry of Science, Research and the Arts Baden-Württemberg (MWK) within the framework of LIBIS/de.NBI Freiburg.
```

Author: maybe its me bt the long one also does not work for me.

Author: ok, apparently it is just me. If it works for the both of you, just add it :)

  Additional funding for projects, as well as the provision of material expenses, is welcome to support our growing Galaxy community in Freiburg and Europe.
  Please also cite the [main Galaxy publication](https://doi.org/10.1093/nar/gkac247).

---

# <i class="fa fa-table"></i> Data Analysis for any Scientist

> **No programming knowledge** is required to access Galaxy and its tools with a web browser. It provides powerful data analysis and can replace Excel habits.

Reviewer: ```suggestion
**No programming knowledge** is required to access Galaxy and its tools; all you need is a web browser. It provides robust data analysis and can replace Excel habits.
```
